packages/ckeditor5-image/docs/_snippets/features/image-resize-buttons-dropdown.html

```diff
@@ -1,5 +1,5 @@
 <div id="snippet-image-resize-buttons-dropdown">
-	<h3>Resize me using the dropdown!</h3>
+	<p>Click me and resize using the pop-up menu dropdown!</p>
```

Comment:
Terminology, here and somewhere else: what you call the "pop-up menu" is what is called the "contextual toolbar" in CKEditor 5 world. As a general UI element it's linked to the feature itself (so: in this case, it's the "image contextual toolbar") and it replaces dialog windows of the past.